bundle/src/main/java/com/adobe/acs/commons/email/impl/EmailServiceImpl.java

```diff
@@ -123,6 +136,72 @@
         return failureList;
     }
 
+    @Override
+    public List<InternetAddress> sendEmail(String templatePath, Map<String, String> emailParams, Map<String, DataSource> attachments, InternetAddress... recipients) {

```

Comment:
+1 lots of copy/pasta otherwise. Ill merge this and make an issue to consolidate.